src/TimeWarpSynchronousGVTManager.hpp

```diff
@@ -51,6 +58,10 @@ class TimeWarpSynchronousGVTManager : public TimeWarpGVTManager {
 
     std::atomic<unsigned int> local_gvt_flag_ = ATOMIC_VAR_INIT(0);
 
+    std::shared_mutex report_gvt_lock_;
+
+    bool report_gvt_;
```

Comment:
initialize the variable here

Reply:
That's the TWSynchronousGVTManager.hpp, did you mean to initialize it into the TWTerminationManager.hpp?